dart/lib/src/event_processor/enricher/web_enricher_event_processor.dart

```diff
@@ -29,6 +30,7 @@ class WebEnricherEventProcessor implements EnricherEventProcessor {
     final contexts = event.contexts.copyWith(
       device: _getDevice(event.contexts.device),
       culture: _getSentryCulture(event.contexts.culture),
+      runtimes: _getRuntimes(event.contexts.runtimes),
```

Comment:
We should also add this in  'html_enricher_event_processor.dart`. Just ran the example app in chrome and wondered why it was not added. Copied it over to test and it worked as expected.

Reply:
Good catch!

Reply:
Wait, I don't think that makes sense. The `html` enricher will never be used for the Flutter version which includes the Flutter runtime information, since the HTML renderer got removed.

Reply:
We diod have a pr where we removed the html renderer, not sure if this is on the main branch or on v9. Depending on that, it would be ok to leave it out. I just had the issue that the new runtime info was not added when i ran the example app with your PR.

Reply:
You need to switch to the master channel of Flutter to test this PR. 

The upcoming Flutter version has no HTML renderer, which means the the html enricher will never be executed when the new Flutter information is actually available. Same thing vice versa, if the HTML enricher gets executed, the Flutter version information is not available since it's an old Flutter version.

This is not dependent on the version of the Sentry SDK, but on the version of the Flutter SDK (but only indirectly, since we don't make use of a Flutter API).

Reply:
Ah that makes sense, sorry for the confusion.